examples/md-flexible/src/configuration/MDFlexConfig.h

```diff
@@ -342,10 +342,11 @@ class MDFlexConfig {
   MDFlexOption<unsigned int, __LINE__> verletRebuildFrequency{
       20, "verlet-rebuild-frequency", true, "Number of iterations after which containers are rebuilt."};
   /**
-   * verletSkinRadius
+   * verletSkinRadiusPerTimeStep
    */
-  MDFlexOption<double, __LINE__> verletSkinRadius{.2, "verlet-skin-radius", true,
-                                                  "Skin added to the cutoff to form the interaction length."};
+  MDFlexOption<double, __LINE__> verletSkinRadiusPerTimestep{
+      .2, "verlet-skin-radius-per-timestep", true,
+      "Skin added to the cutoff to form the interaction length per time step."};
```

Comment:
```suggestion
      "Skin added to the cutoff to form the interaction length. The total skin width is this number times verletRebuildFrequency."};
```
`.2` seems a bit high as a default for this new meaning.